docs/App.Remx.md

```diff
@@ -459,6 +562,61 @@ class PostsList extends Component {
 }
 ...
 ```
+</details>
+<details>
+<summary>With Functional Components</summary>
+
+```js
+import {View, Text, StyleSheet, FlatList} from 'react-native';
+...
+const PostsList = (props) => {
+
+  ...
+  const posts = useConnect(postsStore.getPosts);
+
+  const pushViewPostScreen = post => {
+    Navigation.push(props.componentId, {
+      component: {
+        name: 'blog.ViewPost',
+        passProps: {
+          somePropToPass: 'Some props that we are passing',
+          post
+        },
+        options: {
+          topBar: {
+            title: {
+              text: 'Post1'
+            }
+          }
+        }
+      }
+    });
+  };
+
+  ...
+
+  const renderItem = ({item}) => (
```

Comment:
let's move this out of custom hook

Reply:
did you mean "to custom hook?"..

Reply:
No, I mean this variant:
```tsx
import {View, Text, StyleSheet, FlatList} from 'react-native';
...

const renderItem = ({item}) => (
  <Text onPress={() => pushViewPostScreen(item)}>
    {item.title}
  </Text>
);
const postKeyExtractor = item => `${item.id}-key`;

const PostsList = (props) => {
  ...

```

As these functions do not depend here on anything from props (and often in real life) , they are static ones

Reply:
yeah sure, you're right. though, it won't work for renderItem, because it references pushPostScreen function. moving postKeyExtractor only.